config/src/main/java/com/typesafe/config/impl/Tokenizer.java

```diff
@@ -299,7 +299,7 @@ private Token pullComment(int firstChar) {
         // chars JSON allows to be part of a number
         static final String numberChars = "0123456789eE+-.";
         // chars that stop an unquoted string
-        static final String notInUnquotedText = "$\"{}[]:=,+#`^?!@*&\\";
+        static final String notInUnquotedText = "$\"{}()[]:=,+#`^?!@*&\\";
```

Comment:
I don't remember if something like `foo = (hello)` would parse as `foo = "(hello)"` before, but if so we need it to continue to do so. We should be sure there's a test case for parsing or failing to parse parens in unquoted strings.